test/cbmc/proofs/Sntp_ReceiveTimeResponse/Makefile

```diff
@@ -0,0 +1,35 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+HARNESS_ENTRY = harness
+HARNESS_FILE = Sntp_ReceiveTimeResponse_harness
+
+# Please see test/cbmc/stubs/core_sntp_stubs.c for
+# more information on MAX_NETWORK_RECV_TRIES.
+MAX_NETWORK_RECV_TRIES=5
+
+# Maximum number of sntp time servers
+MAX_NO_OF_SERVERS=5
+
+# Maximum number of iterations needed to receive complete response from server.
+MAX_ITERATIONS_RECEIVE_RESPONSE=2
+
+# This should be a unique identifier for this proof, and will appear on the
+# Litani dashboard. It can be human-readable and contain spaces if you wish.
+PROOF_UID = Sntp_ReceiveTimeResponse
+
+DEFINES +=-DMAX_NO_OF_SERVERS=$(MAX_NO_OF_SERVERS)
+INCLUDES +=
+
+REMOVE_FUNCTION_BODY +=Sntp_DeserializeResponse
+UNWINDSET +=__CPROVER_file_local_core_sntp_client_c_receiveSntpResponse.0:$(MAX_NETWORK_RECV_TRIES)
+UNWINDSET +=__CPROVER_file_local_core_sntp_client_c_Sntp_ReceiveTimeResponse.0:$(MAX_ITERATIONS_RECEIVE_RESPONSE)
```

Comment:
If the unwind set for the loop is set to 2, then doesn't `MAX_ITERATIONS_RECEIVE_RESPONSE` represent only a single iteration to attempt receiving server response?
Am referring to the comment added for the `MAX_ITERATIONS_RECEIVE_RESPONSE` variable